api/features/views.py

```diff
@@ -157,6 +159,41 @@ def get_serializer_context(self):
 
         return context
 
+    @swagger_auto_schema(
+        request_body=FeatureGroupOwnerInputSerializer,
+        responses={200: ProjectFeatureSerializer},
+    )
+    @action(detail=True, methods=["POST"], url_path="add-group-owners")
+    def add_group_owners(self, request, *args, **kwargs):
+        feature = self.get_object()
+        data = request.data
+
+        if not UserPermissionGroup.objects.filter(
+            id__in=data["group_ids"],
+            organisation_id=feature.project.organisation_id,
+        ).count() == len(data["group_ids"]):
+            raise serializers.ValidationError("Some groups not found")
+
+        serializer = FeatureGroupOwnerInputSerializer(data=data)
+        serializer.is_valid(raise_exception=True)
```

Comment:
We should just switch these around since, otherwise, if someone posts an empty body (or any body missing the `group_ids` key) we would throw a 500 instead of, in the code below, a 400 explaining what the issue is. 

```suggestion
        serializer = FeatureGroupOwnerInputSerializer(data=data)
        serializer.is_valid(raise_exception=True)

        if not UserPermissionGroup.objects.filter(
            id__in=serializer.validated_data["group_ids"],
            organisation_id=feature.project.organisation_id,
        ).count() == len(serializer.validated_data["group_ids"]):
            raise serializers.ValidationError("Some groups not found")
```

Reply:
Apologies, the diff looks bad, but all I've done is switched the serializer validation and the ORM validation around (and used `serializer.validated_data` instead of `data["group_ids"]`)

Reply:
Ok cool, I see what you're going for. I've updated the PR with the changes.